src/charm.py

```diff
@@ -768,8 +768,17 @@ def _on_config_changed(self, _):
                     "Duplicate route found; cannot add ingress. Run juju debug-log for details."
                 )
                 return
+        self.unit.set_workload_version(self._get_kubernetes_library_version())
```

Comment:
Should any tests be modified to ensure the version shows up as expected in the status message?

Reply:
Good catch. Unit tests updated.